samples/drivers/led_max7219/prj.conf

```diff
@@ -0,0 +1,10 @@
+# nothing here
+CONFIG_NO_OPTIMIZATIONS=y
+
+CONFIG_SPI=y
+CONFIG_SPI_NRFX=y
+
+CONFIG_SPI_1=y
+CONFIG_SPI_1_NRF_SPI=y
+
+#CONFIG_TRUSTED_EXECUTION_NONSECURE=y
```

Comment:
Remove this line, it is, anyway, commented out

Reply:
Done